### DISCUSSION TITLE
Add option to disable globalMiddlewares in one query/resolver

### DISCUSSION THREAD

**Discussion by anteqkois** [MichalLytek/type-graphql#1759] [Ideas]
It would be nice to have the option to use globalMiddlewares, but sometimes there can be a case when I want to skip this middleware for a specific query. Now I don't see any option to do it.

We can be inspired by NestJS solution:
```ts
consumer
  .apply(LoggerMiddleware)
  .exclude(
    { path: 'cats', method: RequestMethod.GET },
    { path: 'cats', method: RequestMethod.POST },
    'cats/(.*)',
  )
  .forRoutes(CatsController);
```

Or add new decorator to skip all global middleware or them which will be passed to array arg.

I tried this:
```
@UseMiddleware([]) //  to skip default middleware
```

But this don't work

**Comment by MichalLytek** [comment]
I think we could introduce a similar syntax that was designed for the Prisma integration:
https://prisma.typegraphql.com/docs/advanced/additional-decorators

![image](https://github.com/user-attachments/assets/765e31e8-86ae-41c5-9ba4-eabb803a4def)
